app/Domain/Goalcanvas/Controllers/EditCanvasComment.php

```diff
@@ -101,6 +101,7 @@ public function get($params)
             }
 
             $this->tpl->assign('comments', $comments);
+            $this->tpl->assign('id', $canvasItem['id']);
```

Comment:
Few items missing from this file:
- No repo calls (they should all be service calls. If there is no service, just create a wrapper service method
- CANVAS_NAME is still being used